### PR TITLE
filling in missing translations in trainer title desc

### PR DESCRIPTION
**Pull request by gertasik**

### CODE REVIEW

File: src/data/trainer-title-requirements.csv [hunk 1]
```diff
@@ -10,7 +10,7 @@
 "育成でURAファイナルズ決勝に勝利しよう","Win the URA Finals in training"
 "育成で八大競走を全て勝利しよう","Win all eight major races in training"
 "クレーンゲームでぬいぐるみを\n10個以上獲得してクリアしよう","Win 10 or more stuffed toys in the Crane Game"
-"かけがえのない絆を\n感じるひとときを過ごそう","Spend a moment to feel an irreplaceable bond"
+"かけがえのない絆を\n感じるひとときを過ごそう","Win onsen tickets in the lottery, and then win URA Finals"
```
Comment (FabulousCupcake) [inline src/data/trainer-title-requirements.csv:13]:
```suggestion
"かけがえのない絆を\n感じるひとときを過ごそう","Win onsen tickets in the lottery, and then complete training"
```

Comment (FabulousCupcake) [reply src/data/trainer-title-requirements.csv:13]: We'll get another scenario in the future so it's probably good to be vague

File: src/data/trainer-title-requirements.csv [hunk 2]
```diff
@@ -177,4 +177,4 @@
 "ビワハヤヒデのファン数を\n累計1000万人獲得しよう","Reach 10 million total fans for Biwa Hayahide"
 "ビワハヤヒデのファン数を\n累計5000万人獲得しよう","Reach 50 million total fans for Biwa Hayahide"
 "ビワハヤヒデのファン数を\n累計1億人獲得しよう","Reach 100 million total fans for Biwa Hayahide"
-"イベント期間限定ミッションを\nすべてクリアしよう",""
+"イベント期間限定ミッションを\nすべてクリアしよう","Clear all event limited missions"
```
Comment (FabulousCupcake) [inline src/data/trainer-title-requirements.csv:180]:
```suggestion
"イベント期間限定ミッションを\nすべてクリアしよう","Clear all GENTEI MISSION"
```

File: src/data/trainer-title-requirements.csv (hunk 1 above)
Comment (gertasik) [reply src/data/trainer-title-requirements.csv:13]: That really depends on whether or not other scenarios will have the onsen scene, or if they'll even have the lottery - but sure, we can go with this for now

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank

**Pull request merged**